pkg/kn/commands/service/export.go

```diff
@@ -230,3 +302,13 @@ func revisionListSortFunc(revisionList *servingv1.RevisionList) func(i int, j in
 		return a.Name > b.Name
 	}
 }
+
+func stripGeneratedFieldsfromService(svc *servingv1.Service) {
```

Comment:
Can we rename it to `stripIgnoredAnnotationsFromService` and use an array `IGNORED_SERVICE_ANNOTATIONS` (see comment below) ?